src/pmdas/farm/help

```diff
@@ -0,0 +1,705 @@
+Number of read commands from 75-100% of LBA space for the last 3 SMART summary frames
+
+@farm.ata.workload_statistics.write_commands_0_3_lba_space_last_3_smart_summary_frames Write commands 0-3.125% LBA space
+Number of write commands from 0-3.125% of LBA space for the last 3 SMART summary frames
+
+@farm.ata.workload_statistics.write_commands_3_25_lba_space_last_3_smart_summary_frames Write commands 3.125-25% LBA space
+Number of write commands from 3.125-25 % of LBA space for the last 3 SMART summary frames
+
+@farm.ata.workload_statistics.write_commands_25_75_lba_space_last_3_smart_summary_frames Write commands 25-75% LBA space
+Number of write commands from 25-75% of LBA space for the last 3 SMART summary frames
+
+@farm.ata.workload_statistics.write_commands_75_100_lba_space_last_3_smart_summary_frames Write commands 75-100% LBA space
+Number of write commands for 75-100% of LBA space for the last 3 SMART summary frames
+
+@farm.ata.error_statistics.unrecoverable_read_errors Unrecoverable read errors
+Number of unrecoverable read errors on the drive
+
+@farm.ata.error_statistics.unrecoverable_write_errors Unrecoverable write errors
+Number of unrecoverable write errors on the drive
+
+@farm.ata.error_statistics.reallocated_sectors Reallocated sectors
+Number of reallocated sectors on the drive
+
+@farm.ata.error_statistics.read_recovery_attempts Read recovery attempts
+Number of read recovery attempts recorded on the drive
+
+@farm.ata.error_statistics.mechanical_start_failures Mechanical start failures
+Number of mechanical start failures for the drive
+
+@farm.ata.error_statistics.reallocated_candidate_sectors Reallocated candidate sectors
+Number of reallocated candidate sectors for the drive
+
+@farm.ata.error_statistics.asr_events ASR events count
+Number of ASR events for the drive
+
+@farm.ata.error_statistics.interface_crc_errors CRC error count
+Number of interface CRC errors for the drive
+
+@farm.ata.error_statistics.spin_retry_count Spin retry count
+Spin retry count for the drive
+
+@farm.ata.error_statistics.spin_retry_count_normalized Spin retry count normalized
+Spin retry count for the drive normalized
+
+@farm.ata.error_statistics.spin_retry_count_worst Spin retry count worst
+Spin retry count worst for the drive
+
+@farm.ata.error_statistics.ioedc_errors IOEDC error count 
+Number of IOEDC errors (Raw)
+
+@farm.ata.error_statistics.cto_count_total CTO count
+CTO count total for the drive
+
+@farm.ata.error_statistics.cto_count_over_5s CTO count over 5s
+CTO count over 5s for the drive
+
+@farm.ata.error_statistics.cto_count_over_7s CTO count over 7.5s
+CTO count over 7.5s for the drive
+
+@farm.ata.error_statistics.total_flash_led_assert_events Total Flash LED (Assert) events for the drive
+@farm.ata.error_statistics.index_of_last_flash_led Index of the last Flash LED for the drive
+@farm.ata.error_statistics.flash_led_event_0.event_information Flash LED Event 0: Event information
```

Comment:
Could this instead be a single metric with a flash_led_event instance domain? (with instances event_0 through event_7). This is more efficient (e.g. less ondisk space used by pmlogger) and more flexible if additional events come along later.